GitVersionCore/BranchConfigurationCalculator.cs

```diff
@@ -117,8 +131,8 @@ public class BranchConfigurationCalculator
             else
                 errorMessage = "Failed to inherit Increment branch configuration, ended up with: " + string.Join(", ", possibleParents.Select(p => p.Name));
 
-            var hasDevelop = repository.Branches["develop"] != null;
-            var branchName = hasDevelop ? "develop" : "master";
+            var developBranch = repository.Branches.FirstOrDefault(b => Regex.IsMatch(b.Name, "^develop", RegexOptions.IgnoreCase));
```

Comment:
I think this can be removed. The new logic should allow not having this fallback. Let me look into it

Sent from my Windows Phone
________________________________
From: Geert van Horrik<mailto:notifications@github.com>
Sent: ‎03/‎06/‎2015 21:03
To: ParticularLabs/GitVersion<mailto:GitVersion@noreply.github.com>
Cc: Jake Ginnivan<mailto:jake@ginnivan.net>
Subject: Re: [GitVersion] Failing inheritance fixes (#456)


In GitVersionCore/BranchConfigurationCalculator.cs<https://github.com/ParticularLabs/GitVersion/pull/456#discussion_r31663848>:

> @@ -117,8 +131,8 @@ public class BranchConfigurationCalculator
>              else
>                  errorMessage = "Failed to inherit Increment branch configuration, ended up with: " + string.Join(", ", possibleParents.Select(p => p.Name));
>
> -            var hasDevelop = repository.Branches["develop"] != null;
> -            var branchName = hasDevelop ? "develop" : "master";
> +            var developBranch = repository.Branches.FirstOrDefault(b => Regex.IsMatch(b.Name, "^develop", RegexOptions.IgnoreCase));


Then we should use it :)

—
Reply to this email directly or view it on GitHub<https://github.com/ParticularLabs/GitVersion/pull/456/files#r31663848>.